.github/container/test-maxtext.sh

```diff
@@ -13,6 +13,7 @@ usage() {
     echo ""
     echo "  OPTIONS                    DESCRIPTION"
     echo "  -a, --additional-args      Additional fiddle args to pass to MaxText/train.py"
+    echo "  --model-name               Specify model to run. Example: llama2-7b, default"
```

Comment:
You also need to add this to the `getopt` call below; the CI jobs are failing with `getopt: unrecognized option '--model-name'`

Reply:
fixed it